src/docs/sdk/cache.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Cache
+sidebar_order: 20
+---
+
+The SDK's caching integration is agnostic to the underlying engine used. In most cases, the SDK will instrument existing cache abstractions in certain frameworks or libraries.
+
+## Span conventions
+
+### Span Operations
+
+| Span OP | Description |
+|:--|:--|
+| `cache.get` | A single item or multiple items are retrieved from the cache |
+| `cache.put` | A single item or multiple items are written to the cache |
+| `cache.remove` | A single item or multiple items are removed from the cache |
+| `cache.flush` | The entire content of the cache is deleted |
+
+### Span Data
+
+| Data Key | Type | Description | Conditions
+|:--|:--|:--|:--|
+| `cache.hit` | bool | cache hit or miss | required only on read-operations (`cache.get`)
+| `cache.key ` | array | the key(s) involved in the operation | required only on operations that have a key
+
+The following data attributes are only to be set if exposed by the instrumented cache abstraction or retrievable with minimal overhead.
+
+| Data Key | Type | Description | Conditions
+|:--|:--|:--|:--|
+| `cache.item_size ` | int | the size of the item/items read/written/deleted in `bytes` | only on operations that have a key
+| `cache.ttl ` | int | the time to life in `seconds` | only on operations that have a key
+| `network.peer.address ` | string | The hostname of the cache instance |
+| `network.peer.port ` | int | the port used by the cache instance | 
+
+Additionally, the `span.status` can be used to signal if the operation was successful (`success`) or failed (`error`).
```

Comment:
I'm not sure we can actually use `span.status`, as it was tightly coupled with the HTTP status. TBD, else we'll use `cache.success`.

Reply:
The `errro` status is not yet defined here: https://develop.sentry.dev/sdk/event-payloads/span/ 
Do we need to tell anyone on ingest side if we use the new `errror` status?

Reply:
Yep, I'll clarify this